config/routes.rb

```diff
@@ -3,6 +3,19 @@
     namespace :v1 do
       resources :runs, only: [:index, :show, :create, :update, :destroy]
       resources :users, only: [:index, :show, :create, :update, :destroy]
+      get '/users/:user_id/runs', to: 'user_runs#index'
+      get '/users/:user_id/runs/:run_id', to: 'user_runs#show'
+      delete '/users/:user_id/runs/:run_id', to: 'user_runs#destroy'
+      put '/users/:user_id/runs/:run_id', to: 'user_runs#update'
+      post '/users/:user_id/runs', to: 'user_runs#create'
+
+      # scope module: 'users' do
```

Comment:
We had to hand roll these routes because we could not get it to work as we went through and decided to move on and continue working. This is something we can come back and change later on if we feel the need.